### PR TITLE
podgen 0.4.0 (new formula)

### PR DESCRIPTION
**Pull request by tyrchen**
podgen is a static podcast site generator.

### CODE REVIEW

**Comment by dunn** [comment]
Getting a SHA mismatch:
```
==> Downloading from https://codeload.github.com/tyrchen/podgen/tar.gz/v0.4.0
Warning: Formula reports different SHA256: c05c70ae79c437c31cafa0a8d38e7d42636d66d10d804b18480d329eb890e84b
Downloaded to: /Library/Caches/Homebrew/podgen-0.4.0.tar.gz
SHA1: f952f34a3b41a61eb97c337f2466a7b89b2c4443
SHA256: 10c2159efcacef8db0f0507b738b3938fa4a662949a53cbf1737c6d73a040a66
```
Also a bit of whitespace to chomp:
```
== /usr/local/Library/Formula/podgen.rb ==
C: 86:  1: 1 trailing blank lines detected.

1 file inspected, 1 offense detected

**Comment by tyrchen** [comment]
Sorry for failing the CI again. I checked the log, but didn't find any useful information. Can anyone help to point out what was the issue this time? Thanks a lot!

File: Library/Formula/podgen.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    mkdir_p buildpath/"src/github.com/tyrchen/"
+    ln_sf buildpath, buildpath/"src/github.com/tyrchen/podgen"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", bin/"podgen", "main.go"
+  end
+
+  test do
+    File.write("items.yml", "")
```
Comment (DomT4) [inline Library/Formula/podgen.rb]: This caused your error:

```
Error: podgen: failed
undefined method `write' for File:Class
/usr/local/Library/Formula/podgen.rb:81:in `test'
```

Use `(testpath/"items.yml").write ""` instead. Should work fine.

Comment (DomT4) [inline Library/Formula/podgen.rb]: Or even just `touch "items.yml"` maybe?

Comment (DomT4) [inline Library/Formula/podgen.rb]: Ping?

**Comment by bfontaine** [comment]
Ping?

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit as fresh with the comments above addressed, if you'd like to. Thanks for the submission to Homebrew!

**Pull request closed without merging**